backend/uclapi/marketplace/views.py

```diff
@@ -1,5 +1,5 @@
 from django.shortcuts import render
 
 
-def marketplace(request):
+def marketplace(request, id=None):
```

Comment:
This requires a comment, on most linters this will pick up as an unused variable.

Reply:
So we don't actually need to pass the id to react because react router already has it. But django needs to have the path /marketplace/:appid defined but not do anything with the id, when you say requires a comment shall I just explain that over the top or change it?

Reply:
Yes, the explanation that you've given now needs to be in a comment above the function definition (modulo the grammar errors).

Reply:
Haha! Okay I will add that, thanks!